package.json

```diff
@@ -55,9 +55,9 @@
     "bcurl": "0.1.4",
     "bfile": "~0.1.3",
     "binet": "~0.3.3",
-    "bledger": "bcoin-org/bledger",
+    "bledger": "~0.1.4",
```

Comment:
Its now `~0.1.5`
https://github.com/bcoin-org/bledger/commit/63d65d3d43d088af3637cfb1970104638b3c9d7c

Reply:
haha, good looking out. That looks like it was in the past 10 minutes since I pushed!